shared/chat/dumb.js

```diff
@@ -100,7 +100,6 @@ const commonConvoProps = {
   isRequesting: false,
   onPostMessage: (text: string) => console.log('on post', text),
   selectedConversation: 'convo1',
-  emojiPickerOpen: false,
```

Comment:
we disabled rendering this and it shouldn't be a prop anymore, more of a local state